src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.js

```diff
@@ -11,11 +11,12 @@ import HiddenInput from 'signals/incident-management/components/HiddenInput';
 
 import { ChevronDown, ChevronUp } from '@datapunt/asc-assets';
 import { Button } from '@datapunt/asc-ui';
+import { ascDefaultTheme } from '@datapunt/asc-core';
 
 import './style.scss';
 
 const StyledButton = styled(Button)`
-  border: 1px solid black;
+  border: 1px solid ${ascDefaultTheme.colors.tint.level7};;
```

Comment:
Same remark applies here as for the colour `white`; `tint.level7` is a very generic indicator and might change over time which might not be what we want/need.

Reply:
gedaan